lab1.js

```diff
@@ -134,7 +158,21 @@ var tooHungryDay;
 */
 
 // your code goes here
+var meals = 0;
+tooHungryDay = -1;
+
+for (var i = 0; i < mealsPerDay.length; i++) {
+  meals = meals + mealsPerDay[i];
+  var avg = meals / (i + 1);
+
+  console.log('Day ' + (i + 1) + ' average meals per day that lion got since new caretaker started is ' + avg);
 
+  if (avg < 4 && tooHungryDay < 0) {
```

Comment:
good job checking to see if you're already assigned a value to tooHungryDay. You can also use a break at the end of this if block to end execution of the for loop.